Makefile

```diff
@@ -1,21 +1,54 @@
 # Watson
 
+ifndef PYTHON
+	PYTHON = python
+endif
+
+ifndef PIP
+	PIP = pip
+endif
+
+VENV = virtualenv
+VENV_ARGS = -p $(PYTHON)
+VENV_DIR = $(CURDIR)/.env
+
```

Comment:
These Makefile change should be in an extra PR (together with the .gitignore change above).